src/main/java/net/neoforged/neoforge/client/model/data/ModelDataManager.java

```diff
@@ -109,6 +129,11 @@ private boolean isOtherThread() {
         }
     }
 
+    /**
+     * A snapshot of the state of the active manager at the point in time when a chunk section was prepared for re-rendering.
```

Comment:
Should probably clarify that it's not a snapshot of the *entire* active manager (although I should hope most people can figure this out from a cursory code read).